src/aleph_client/commands/utils.py

```diff
@@ -291,28 +364,42 @@ async def update_table(queue: asyncio.Queue, table: Table):
             break
         table.add_row(*data)
 
-async def fetch_crn_system(session: aiohttp.ClientSession, node: NodeInfo) -> str:
+async def fetch_crn_system(session: aiohttp.ClientSession, node: NodeInfo) -> CpuProperties:
+    data = None
+
     try:
         async with async_timeout.timeout(settings.HTTP_REQUEST_TIMEOUT + settings.HTTP_REQUEST_TIMEOUT * 0.3 * random()):
             url: str = node["address"].rstrip('/') + '/about/usage/system'
             async with session.get(url) as resp:
                 resp.raise_for_status()
-                data = await resp.json()
+                data_raw = await resp.json()
+                data = MachineUsage.parse_obj(data_raw)
+                # print("data:", data)
+    except TimeoutError:
+        logger.debug(f'Timeout while fetching: {url}')
+    except aiohttp.client_exceptions.ClientConnectionError:
+        logger.debug(f'Error on connection: {url}')
     except Exception as e:
-        data = None
+        logger.debug(f'This error occured: {e}')
     return data
 
 async def get_crn_version(session: aiohttp.ClientSession, node: NodeInfo) -> str:
-    version = None
+    url = node["address"]
+    version = "Can't fetch the version"
```

Comment:
Why use a hardcoded string instead of either returning `Optional[str]` or raise an exception ?

Reply:
This was a temporary state for the version, I just wanted to check if the fetch of the version worked, and the first thing that came to mind was to put this.

My code needs something in the version field to be able to put it in the tab. So raising an exception, or putting it as optional, wouldn't work for the current version.

I intend to change it in the future,